drivers/modem/modem_socket.c

```diff
@@ -0,0 +1,281 @@
+/** @file
+ * @brief Modem socket / packet size handler
+ *
+ * Generic modem socket and packet size implementation for modem context
+ */
+
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <net/socket_offload.h>
+#include <net/socket_offload_ops.h>
```

Comment:
It seems that socket offloading is handled elsewhere in this design. You could remove the headers to avoid confusion.

Reply:
nice catch.  I attempted to move the offloaded ops into modem_socket at one point and left the headers.